.github/workflows/branches.yml

```diff
@@ -0,0 +1,220 @@
+env:
```

Comment:
I would add an autogenerated comment to this YAML file to suggest this can't be manually edited

Reply:
this isn't possible at the moment because the library doesn't support outputting yaml comments :(